addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -23,6 +25,11 @@ import (
 	cniv1alpha2 "github.com/vmware-tanzu/tanzu-framework/apis/addonconfigs/cni/v1alpha2"
 )
 
+const (
+	bootstrapFromInline            = "Inline"
+	bootstrapFromSupervisorCluster = "SupervisorCluster"
+)
+
 // AntreaConfigSpec defines the desired state of AntreaConfig
 type AntreaConfigSpec struct {
```

Comment:
AntreaConfigSpec should be aligned with Antrea package schema.yaml.

The AntreaNSX member in this struct needs to be renamed as antrea_nsx, and ``yaml:"antreaNsx,omitempty"`` needs to be changed to ``yaml:"antrea_nsx,omitempty"``.

Reply:
done.